### PR TITLE
A tiny js world

### PR DESCRIPTION
**Pull request by webdevtoday**
### a-tiny-JS-world

[Demo](https://webdevtoday.github.io/a-tiny-JS-world/) |
[Code base](https://github.com/webdevtoday/a-tiny-JS-world/tree/gh-pages)

The code is submitted in a dedicated feature branch.
Please, review.

### CODE REVIEW

File: submissions/webdevtoday/a-tiny-js-world/index.js [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+   print('human; <strong>John</strong>; male; 2; 2; <em>Hello world!</em>; Rex, Tom, Jenny');
+   print('human; <strong>John</strong>; male; 2; 2; <em>Hello world!</em>; Rex, Tom, Jenny', 'div');
+   */
+
+
+const printList = [dog, cat, man, woman, catwoman];
+
+printList.forEach( obj => {
+   print(`${obj.species};${obj.name};${obj.gender};${obj.legs};${obj.hands};${obj.saying};${obj.friends.join(', ')}`);
+} );
```
Comment (madmaxWMFU) [inline submissions/webdevtoday/a-tiny-js-world/index.js:82]: Create a function which returns a template string

**Review by madmaxWMFU — CHANGES_REQUESTED** [review]
Good job!
You should send the only file with changes. It is a file `index.js`

File: submissions/webdevtoday/a-tiny-js-world/index.js (hunk 1 above)
Comment (webdevtoday) [reply submissions/webdevtoday/a-tiny-js-world/index.js:82]: Ok! I'll do it.

Comment (webdevtoday) [reply submissions/webdevtoday/a-tiny-js-world/index.js:86]: I did it

**Review by madmaxWMFU — APPROVED** [review]

**Comment by webdevtoday** [comment]
@madmaxWMFU Thank you very much!!!

**Comment by madmaxWMFU** [comment]
@MaxKalashnyk can you check?)

**Comment by webdevtoday** [comment]
Ohh, I didn't check that this PR don't close and I send #done to chat((

**Comment by webdevtoday** [comment]
I deleted message

File: submissions/webdevtoday/a-tiny-js-world/index.js [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+   print('human; John; male; 2; 2; Hello world!; Rex, Tom, Jenny');
+   print('human; <strong>John</strong>; male; 2; 2; <em>Hello world!</em>; Rex, Tom, Jenny');
+   print('human; <strong>John</strong>; male; 2; 2; <em>Hello world!</em>; Rex, Tom, Jenny', 'div');
+   */
+
+
+const printList = [dog, cat, man, woman, catwoman];
+
+function tmplStr(obj) {
```
Comment (MaxKalashnyk) [inline submissions/webdevtoday/a-tiny-js-world/index.js:80]: You can destructure `obj` here

**Review by MaxKalashnyk — COMMENTED** [review]
Good job! Look at my minor suggestion and we're ready to merge

File: submissions/webdevtoday/a-tiny-js-world/index.js (hunk 2 above)
Comment (webdevtoday) [reply submissions/webdevtoday/a-tiny-js-world/index.js:80]: Ok! I'll do it

Comment (webdevtoday) [reply submissions/webdevtoday/a-tiny-js-world/index.js]: I did it

**Review by MaxKalashnyk — APPROVED** [review]
LGTM!

**Comment by webdevtoday** [comment]
@MaxKalashnyk Thanks a lot!!!

**Pull request merged**